### PR TITLE
Prepare release v1.28.0

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,38 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-13)
+
+- Bump jsdom from 21.1.0 to 21.1.1 ([@dependabot](https://github.com/dependabot) in [#1257](https://github.com/jaegertracing/jaeger-ui/pull/1257))
```
Comment (albertteoh) [inline CHANGELOG.md:11]: I think we should remove these dependabot changes.

Comment (pavolloffay) [reply CHANGELOG.md]: Removed, although the previous release lists some of them. 

We should document this in the release.md.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,22 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-13)
+
+- Update TypeScript and ESLint to latest versions ([@mszabo-wikia](https://github.com/mszabo-wikia) in [#1256](https://github.com/jaegertracing/jaeger-ui/pull/1256))
```
Comment (albertteoh) [inline CHANGELOG.md:11]: We should curate and categorise these into the respective Enhancements and Fixes sections below.

File: CHANGELOG.md (hunk 1 above)
Comment (albertteoh) [reply CHANGELOG.md]: Yeah, I suspect they might have been relevant to the patch release, though.

File: CHANGELOG.md (hunk 2 above)
Comment (pavolloffay) [reply CHANGELOG.md:11]: I have moved them to fixes, as none of them is a user-visible improvement (as far as I understand).

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,22 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-13)
+
+### Enhancements
```
Comment (yurishkuro) [inline CHANGELOG.md:11]: Missing this one?
```
e2297dc - Make traceID, startTime, endTime, duration and traceName available fo… (#1178) (5 weeks ago) <Kanahathi Mohideen>
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,22 @@
...
+
+### Fixes
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply CHANGELOG.md:15]: Added - see my previous comment

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,33 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-15)
+
+### Enhancements
+
+- Make traceID, startTime, endTime, duration and traceName available fo… ([@MUI-Pop](https://github.com/MUI-Pop) in [#1178](https://github.com/jaegertracing/jaeger-ui/pull/1178))
```
Comment (yurishkuro) [inline CHANGELOG.md:13]:
```suggestion
- Make traceID, startTime, endTime, duration and traceName available for Link Patterns ([@MUI-Pop](https://github.com/MUI-Pop) in [#1178](https://github.com/jaegertracing/jaeger-ui/pull/1178))
```

**Review by yurishkuro — DISMISSED** [review]

**Comment by pavolloffay** [comment]
PR updated with the latest changes in the main. I will merge it to proceed with the release as the PR was already approved.

**Pull request merged**